static/source/guides/getting_started.html.slim

```diff
@@ -42,60 +42,121 @@ section.guide
+          `DANGER_GITHUB_HOST` to the host that GitHub is running on.
 
-      Do not add the bot to your repo or to your organization.
+          `DANGER_GITHUB_API_HOST` to the host that the GitHub Enterprise API is reachable on.
 
-      ##### Closed Source Projects
+          For example:
 
-      Add the bot to repo or to your organization. The bot requires permission level "Write" to be able to set a PR's status. Note that you _should not_ re-use this bot for OSS projects.
+          ```
+          DANGER_GITHUB_API_HOST=https://git.corp.evilcorp.com/api/v3
+          DANGER_GITHUB_HOST=git.corp.evilcorp.com
+          ```
 
-      ### Setting up an Access Token
+          [github_bots]: https://twitter.com/sebastiangrail/status/750844399563608065
+          [github_token]: https://github.com/settings/tokens/new
+        
+      div.gitlab
+        markdown:
 
-      [Here's the link][github_token], you should open this in the private session where you just created the new GitHub account. Again, the rights that you give to the token depend on the openness of your projects.
+          To get the most out of Danger, we recommend giving her the ability to post comments in your code review. This is a regular GitLab account, but depending on whether you are working on a private or public project, you will want to give different levels of access to this bot.
 
-      ##### Tokens for OSS Projects
+          To get started, open [https://gitlab.com](https://gitlab.com) (or your GitLab instance) in a private browser session.
 
-      We recommend giving the token the smallest scope possible. This means just `public_repo`, this scopes limits Danger's abilities to just writing comments on OSS projects. Because the token can be quite easily be extracted from the CI environment, this minimizes the chance for bad actors causing chaos with it.
+          ##### OSS Projects
 
-      ##### Tokens for Closed Source Projects
+          Do not add the bot to your project or to your group.
 
-      We recommend giving access to the whole `repo` scope, and its children.
+          ##### Closed Source Projects
+
+          Add the bot to your private project or to your private group with the "Guest" permission level. Note that you _should not_ re-use this bot for OSS projects. As the access token you use could be extracted and would give access to your closed source projects.
+
+          ### Setting up an Access Token
+
+          [Here's the link][gitlab_token], you should open this in the private session where you have just created the new GitLab account. You'll want to copy the token for later, when you add a `DANGER_GITLAB_API_TOKEN` to your CI.
+
+          ### Self-hosted GitLab
+
+          To let Danger know the API details around your custom setup, you need to
+          set two env variables:
+
+          `DANGER_GITLAB_HOST` to the host that GitLab is running on.
+
+          `DANGER_GITLAB_API_HOST` to the host that the GitLab API is reachable on.
```

Comment:
This one is currently called `DANGER_GITLAB_API_ENDPOINT`, but I am open to changing it. `DANGER_GITLAB_API_BASE_URL` might be better along with `DANGER_GITHUB_API_BASE_URL` for GitHub